src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs

```diff
@@ -62,6 +62,16 @@ internal static ImagePartType GetImagePartType(string extension)
                 ".svg" => ImagePartType.Svg,
                 ".tiff" => ImagePartType.Tiff,
                 ".wmf" => ImagePartType.Wmf,
+                "image/bmp" => ImagePartType.Bmp,
```

Comment:
clever. probably should rename the input argument from "extension" to something more general (?). Not sure what.

Reply:
Also, not sure if this is a good idea. Might be better to have a separate GetImagePartTypeByMime() or something.

Reply:
This seems fine - the API name isn't saying it's from extensions. How are we handling this in v3.0?

Reply:
> should rename the input argument

Probably ok, but that can be a source breaking change if someone did `GetImagePartType(extension: ".something")`

Reply:
> Having one method that handles this seems fine - the API name isn't saying it's from extensions. How are we handling this in v3.0?

v3.0 has the same issue, but the fix will have to be a bit different